src/components/DopplerPlus/AgenciesForm.js

```diff
@@ -100,29 +105,29 @@ const AgenciesForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } })
             <BreadcrumbItem href={_('agencies.breadcrumb_url')} text={_('agencies.breadcrumb')} />
             <BreadcrumbItem text={_('agencies.title')} />
           </Breadcrumb>
-          <h2>Plan para agencias</h2>
-          <p>Gestiona y monitorea las cuentas de todos tus clientes desde un mismo lugar.</p>
+          <h2>{_('agencies.title')}</h2>
+          <p>{<FormattedMessageMarkdown id="agencies.subtitle_MD" />}</p>
         </div>
       </HeaderSection>
 
       <section className="dp-container">
         <div className="dp-rowflex">
           <div className="col-lg-8 col-md-12 m-b-24">
             <div className="dp-wrapper-form-plans">
-              <h3>Completa el siguiente formulario y te contactaremos a la brevedad</h3>
+              <h3>{_('agencies.form_legend')}</h3>
               <FormWithCaptcha
                 onSubmit={onSubmit}
                 initialValues={getFormInitialValues()}
                 validate={validate}
               >
-                <legend>Completa el siguiente formulario de planes</legend>
+                <legend>{_('agencies.form_legend')}</legend>
                 <fieldset>
                   <FieldGroup>
                     <EmailFieldItem
                       autoFocus
                       type="email"
                       fieldName={fieldNames.email}
-                      label="Email:"
+                      label={'*' + _('forms.label_email')}
```

Comment:
How about using string literals here? to avoid the +
```suggestion
                      label={`*${ _('forms.label_email')}`}
```